README.md

```diff
@@ -116,7 +116,7 @@ You are also welcome to join us on the [matrix chatroom](https://matrix.to/#/#tl
 ## Similar projects
 
 - [Command Line Interface Pages](https://github.com/command-line-interface-pages)
-  allows you to write standardized help pages for CLI, directories and configs.
+  allows you to write standardized help pages for CLI, directories, and configs.
 
 - [Cheat](https://github.com/cheat/cheat)
   allows you to create and view interactive cheatsheets on the command-line.
```

Comment:
Hi, I think in line 123 it should be Unix instead of `*nix` (a possible typo).

Reply:
In their repo description, it's `*` too. `*` is near `U` and you need to press shift for both so probably it is a typo.

Reply:
I was searching this usage online, and according to [Stack Overflow](https://stackoverflow.com/questions/4715374/what-is-the-meaning-of-nix), this seems like a valid use case. But IMO a beginner might confuse it with Nix (OS/package manager/programming language), so it is better to refer to UNIX here.